src/window.rs

```diff
@@ -139,6 +139,50 @@ impl Window {
             }
         });
         window.add_action(&fullscreen_action);
+
+        // Zoom in
+        let zoom_action = gtk4::gio::SimpleAction::new("zoom-in", None);
+        let window_weak = self.downgrade();
```

Comment:
What is the window used for?

Reply:
Oops, those were accidental left overs because I copy and pasted an existing action.